e2e/test/scenarios/dashboard/visualizer/basics.cy.spec.ts

```diff
@@ -445,10 +445,44 @@ describe("scenarios > dashboard > visualizer > basics", () => {
 
     // Making sure the card renders
     H.getDashboardCard(0).within(() => {
-      cy.findByText(`Count (${PRODUCTS_COUNT_BY_CREATED_AT.name})`).should(
-        "exist",
+      cy.findAllByText(`Count (${PRODUCTS_COUNT_BY_CREATED_AT.name})`).should(
```

Comment:
Now it is more correct as added series gets its own y-axis with its series name as the y-axis label:

Before
<img width="553" alt="Screenshot 2025-05-06 at 10 43 34 PM" src="https://github.com/user-attachments/assets/56480bbb-81b7-4cfe-bcb8-0a14f750df7b" />

After
<img width="539" alt="Screenshot 2025-05-06 at 10 43 58 PM" src="https://github.com/user-attachments/assets/1e8754e0-3b56-48ed-a102-976946b8a8db" />